jsk_2016_01_baxter_apc/node_scripts/set_tote_contents_param.py

```diff
@@ -17,5 +17,5 @@
     json_file = args.json_file
 
     data = json.load(open(json_file))
-    rospy.set_param('~tote_contents', data['tote_contents'])
+    rospy.set_param('/tote_contents', data['tote_contents'])
```

Comment:
I confirmed that it works.

<img width="1280" alt="1" src="https://cloud.githubusercontent.com/assets/4310419/19356470/ed4abab8-91a8-11e6-86a6-c48a0bed2377.png">
<img width="1280" alt="2" src="https://cloud.githubusercontent.com/assets/4310419/19356475/ef4413c8-91a8-11e6-8e57-d229d08110d7.png">
<img width="1280" alt="3" src="https://cloud.githubusercontent.com/assets/4310419/19356479/f3101d08-91a8-11e6-9a52-630f5fc94646.png">
<img width="1280" alt="4" src="https://cloud.githubusercontent.com/assets/4310419/19356483/f64e4eae-91a8-11e6-9c59-1b0f197a2221.png">
<img width="1280" alt="5" src="https://cloud.githubusercontent.com/assets/4310419/19356488/f8ca0e5c-91a8-11e6-840f-0f3b6fa9b7ae.png">